src/instructlab/generator/generate_data.py

```diff
@@ -81,14 +111,19 @@
 ]
 
 
-def check_prompt_file(prompt_file_path):
+def check_prompt_file(prompt_file_path, model_family):
     """Check for prompt file."""
     try:
         with open(prompt_file_path, encoding="utf=8") as file:
             prompt_template = file.read()
     except FileNotFoundError:
-        print(f"Cannot find {prompt_file_path}. Using default prompt.")
-        prompt_template = DEFAULT_PROMPT_TEMPLATE
+        print(
+            f"Cannot find {prompt_file_path}. Using default prompt depending on model-family."
+        )
+        if model_family == "merlinite":
+            prompt_template = DEFAULT_PROMPT_TEMPLATE_MERLINITE
+        elif model_family == "mixtral":
+            prompt_template = DEFAULT_PROMPT_TEMPLATE_MIXTRAL
```

Comment:
Include an error case so an unknown family does not cause a NameError:
```suggestion
            prompt_template = DEFAULT_PROMPT_TEMPLATE_MIXTRAL
        else:
            raise ValueError(f"Unsupported family '{model_family}'")
```